kivy/tools/packaging/pyinstaller_hooks/__init__.py

```diff
@@ -64,10 +64,14 @@
 import logging
 from os.path import dirname, join
 import importlib
+import subprocess
+import re
+import glob
 
 import kivy
 import kivy.deps
 from kivy.factory import Factory
+from PyInstaller.depend import bindepend
```

Comment:
Are you sure about this?
Because it is failing with travis currently:
https://travis-ci.org/kivy/kivy/jobs/494325672

Reply:
The purpose of that import is to find all the libraries that the GStreamer plugins link to, which otherwise aren't included in the package.

I just committed a change to .travis.yml that gets those jobs working again - how does that look? I see that a couple of the jobs are still failing, but are also failing on master.